cmd/flipt/main.go

```diff
@@ -639,13 +648,26 @@ func run(ctx context.Context, logger *zap.Logger) error {
 
 		logger.Debug("starting http server")
 
-		color.Green("\nAPI: %s://%s:%d/api/v1", cfg.Server.Protocol, cfg.Server.Host, httpPort)
+		var (
+			apiAddr = fmt.Sprintf("%s://%s:%d/api/v1", cfg.Server.Protocol, cfg.Server.Host, httpPort)
+			uiAddr  = fmt.Sprintf("%s://%s:%d", cfg.Server.Protocol, cfg.Server.Host, httpPort)
+		)
 
-		if cfg.UI.Enabled {
-			color.Green("UI: %s://%s:%d", cfg.Server.Protocol, cfg.Server.Host, httpPort)
-		}
+		if isConsole {
+			color.Green("\nAPI: %s", apiAddr)
+
+			if cfg.UI.Enabled {
+				color.Green("UI: %s", uiAddr)
+			}
+
+			fmt.Println()
+		} else {
+			logger.Info(fmt.Sprintf("api: %s", apiAddr))
```

Comment:
Take it or leave it:
```suggestion
			logger.Info("api starting", zap.String("address", apiAddr))
```